src/libraries/Common/tests/TestUtilities/System/PlatformDetection.cs

```diff
@@ -226,17 +226,17 @@ private static bool GetIsInContainer()
             if (IsWindows)
             {
                 string key = @"HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Control";
-                string value = "";
+                object value = null;
 
                 try
                 {
-                    value = (string)Registry.GetValue(key, "ContainerType", defaultValue: "");
+                    value = Registry.GetValue(key, "ContainerType", defaultValue: null);
```

Comment:
Nit:

```suggestion
                    return Registry.GetValue(key, "ContainerType", defaultValue: null) is not null;
```

Not sure why we keep this in a try/catch block? Which type of exception is it compensating for?

Cf. https://github.com/dotnet/runtime/pull/42062/commits/5f845f6f1e1525f91127da6c6b1de3be1cae95fe#diff-9bc9aae45b52414dacb7f1ab20dfb587L62

Reply:
I'm not sure which type of exception it is compensating for. I'll update it.